libs/hash/definitions/hashLib.h

```diff
@@ -0,0 +1,15 @@
+#include "openssl/crypto.h"
+#include "openssl/sha.h"
+
+#ifndef __HASHLIB_H
+#define __HASHLIB_H
+
+#define HASH_SIZE SHA256_DIGEST_LENGTH
+
+typedef unsigned char HASH[HASH_SIZE];
+
+void calcHash(int val, HASH hash);
+
+void printHash(unsigned char hash[]);
```

Comment:
suggestion:  you can use only HASH type as you created before.

Reply:
fixed at [db63c5d](https://github.com/jowjow22/simplified-c-blockchain/pull/5/commits/db63c5d99982e1a4d2423f845fa9488f2f06a8ac)